cfme/common/vm.py

```diff
@@ -991,18 +1011,20 @@ def _select_rows(indexes):
 
 
 @attr.s
-class VMCollection(BaseVMCollection):
+class VMCollection(BaseVMCollection, RetirementMixin):
     ENTITY = VM
 
-    def retire(self, entities):
+    def retire(self, **kwargs):
         """Select and then retire one or more VMs|Instances from the All VMs|Instances page.
 
         Args:
```

Comment:
Since `entities` is now a `kwarg` this should be changed to `Kwargs`

Reply:
Changed.